resources/public/api/conf/3.0/common/headers.yaml

```diff
@@ -0,0 +1,53 @@
+components:
+  parameters:
+    acceptJson:
+      name: "Accept"
+      in: header
+      description: |
+        Specifies the response format and the version of the API to be used.
+      schema:
+        type: string
+        enum:
+          - "application/vnd.hmrc.3.0+json"
+      required: true
+
+    authorization:
```

Comment:
best to split `authorization ` into `authorizationReadScopeHeader` and `authorizationWriteScopeHeader` like is done [here](https://github.com/hmrc/individuals-expenses-api/blob/main/resources/public/api/conf/1.0/common/headers.yaml) and use as appropriate to each endpoint